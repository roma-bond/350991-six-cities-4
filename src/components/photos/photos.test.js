import React from "react";
import renderer from "react-test-renderer";
import Photos from "./photos.jsx";

const offer = {
  premium: true,
  price: 120,
  name: `Beautiful & luxurious apartment at great location`,
  type: `Apartment`,
  rating: 4,
  photoSrc: [`img/apartment-01.jpg`]
};

it(`Should Photos render correctly`, () => {
  const tree = renderer
    .create(<Photos
      photos={offer.photoSrc}
      type={offer.type}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
