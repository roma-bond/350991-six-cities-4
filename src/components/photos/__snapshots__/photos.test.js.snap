// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Photos render correctly 1`] = `
<div
  className="property__image-wrapper"
>
  <img
    alt="Photo Apartment"
    className="property__image"
    src="img/apartment-01.jpg"
  />
</div>
`;
